### PR TITLE
March Into Mapness - Mass driver router

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

Adds a new space ruin; the Mass-driver router.
Contains 4 mass drivers in cardinal directions, for quick space travel!
Also has a fake emag, medkit, and two toolboxes

Also alphabetically sorts the ruin blacklist
### Mapping March
Ckey to receive rewards: jimmyl

## Why It's Good For The Game

Wooshing through space is cool i think

## Changelog
:cl:
add: A new ruin has drifted into the horizon, the Mass driver router
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -356,3 +356,9 @@
 	suffix = "the_faceoff.dmm"
 	name = "The Faceoff"
 	description = "What do you get when a meeting of the enemy corporations get crashed?"
+
+/datum/map_template/ruin/space/the_faceoff
```
Comment (san7890) [inline code/datums/ruins/space.dm:360]: i think you copypasted a bit too hard, make sure its a unique subtype

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by Hermod132** [comment]
This is an L for me, I had a very similar idea to this in a nearly completed state. I'll prob still push what I was working on anyways.

File: code/datums/ruins/space.dm (hunk 1 above)
Comment (mc-oofert) [reply code/datums/ruins/space.dm:360]: i am so stupid

**Comment by Addust** [comment]
make it feel like an old cargo routing post - goonstation has mass-driver-based cargo (because their shuttle system is dogshit) and this could be used to reference that - it shouldn't be large but it could use an abandoned crate or two

**Comment by mc-oofert** [comment]
> I think #74129 is the "better evolution" of the concept, that is to say it's more visually interesting and a more logical ruin than just a platform in space. Love the concept tho
> 
> I'm gonna DMN this in favor of that but if a maptainer opposes me they're free to remove it

both of the PRs were speedran as soon as I got and said the idea in mapping general...

gonna try to rework my one so dont close this yet

**Comment by Addust** [comment]
this looks a lot better, maybe add a few more RTGs and make sure they're advanced?

**Comment by mc-oofert** [comment]
> this looks a lot better, maybe add a few more RTGs and make sure they're advanced?

1 advanced rtg is already enough

**Comment by Addust** [comment]
based, laddie
image

**Comment by Addust** [comment]
(i know it won't have any effect ingame but forgottenship is based and needs to be cherished once a way to bypass them becoming discount nukies is found)

**Comment by mc-oofert** [comment]
third merge conflict moment

**Review by Maurukas — APPROVED** [review]

**Pull request merged**